src/components/TitleCard/index.js

```diff
@@ -21,8 +22,13 @@ export default ({
         {intro}
       </Maybe>
     </div>
-    <Card>
-      {children}
-    </Card>
+    {subtle
+      ? <div className='bt b--gray-secondary'>
+          {children}
+        </div>
+      : <Card>
+          {children}
+        </Card>
+    }
```

Comment:
This allows the new `subtle` option for my local `TitleCard` (not in egghead-ui, only `Card` is in egghead-ui currently).
![image](https://cloud.githubusercontent.com/assets/5497885/24826269/de33571c-1bee-11e7-8aa7-3d01d1667427.png)